test/KillSwitchOracleIntegration.t.sol

```diff
@@ -76,212 +79,79 @@ contract KillSwitchOracleIntegrationTest is Test {
-            frozen:               false,
-            paused:               false,
-            ltv:                  0,
-            liquidationThreshold: 75_00,
-            liquidationBonus:     107_00
-        });
-        _assertReserve({
-            asset:                GNO,
-            active:               true,
-            frozen:               true,
-            paused:               false,
-            ltv:                  0,
-            liquidationThreshold: 25_00,
-            liquidationBonus:     110_00
-        });
-        _assertReserve({
-            asset:                RETH,
-            active:               true,
-            frozen:               false,
-            paused:               false,
-            ltv:                  0,
-            liquidationThreshold: 79_50,
-            liquidationBonus:     107_00
-        });
-        _assertReserve({
-            asset:                USDT,
-            active:               true,
-            frozen:               true,
-            paused:               false,
-            ltv:                  0,
-            liquidationThreshold: 0,
-            liquidationBonus:     0
-        });
-    }
-
-    function _assertReserve(
-        address asset,
-        bool active,
-        bool frozen,
-        bool paused,
-        uint256 ltv,
-        uint256 liquidationThreshold,
-        uint256 liquidationBonus
-    ) internal {
-        DataTypes.ReserveConfigurationMap memory configuration = pool.getConfiguration(asset);
+        assertEq(_getBorrowEnabled(DAI),    false);
+        assertEq(_getBorrowEnabled(SDAI),   false);
+        assertEq(_getBorrowEnabled(USDC),   false);
+        assertEq(_getBorrowEnabled(WETH),   false);
+        assertEq(_getBorrowEnabled(WSTETH), false);
+        assertEq(_getBorrowEnabled(WBTC),   false);
+        assertEq(_getBorrowEnabled(GNO),    false);
+        assertEq(_getBorrowEnabled(RETH),   false);
+        assertEq(_getBorrowEnabled(USDT),   false);
+
+        // Test the functionality of the pool
+        uint256 userBalance = 40_187_695.578838876771725671e18;
+        deal(DAI, DAI_BORROWER_WALLET, 1e18);
+        assertEq(IERC20(DAI_VAR_DEBT).balanceOf(DAI_BORROWER_WALLET), userBalance);
         
-        assertEq(configuration.getActive(),               active);
-        assertEq(configuration.getFrozen(),               frozen);
-        assertEq(configuration.getPaused(),               paused);
-        assertEq(configuration.getLtv(),                  ltv);
-        assertEq(configuration.getLiquidationThreshold(), liquidationThreshold);
-        assertEq(configuration.getLiquidationBonus(),     liquidationBonus);
+        vm.startPrank(DAI_BORROWER_WALLET);
+
+        // Make sure we can repay
+        IERC20(DAI).approve(address(pool), 1e18);
+        pool.repay(DAI, 1e18, 2, DAI_BORROWER_WALLET);
+
+        // Borrow should revert on all assets
+        vm.expectRevert(bytes('30'));  // BORROWING_NOT_ENABLED
```

Comment:
Can import Errors and do `bytes(Errors.BORROWING_NOT_ENABLED)`

Reply:
We should add SDAI and GNO here just to be comprehensive

Reply:
fixed

Reply:
missed the top part - also done now